src/ngraph/runtime/cpu/mkldnn_utils.hpp

```diff
@@ -24,6 +24,76 @@
 #include "ngraph/runtime/cpu/op/batch_norm_relu.hpp"
 #include "ngraph/type/element_type.hpp"
 
+#if MKLDNN_VERSION_MAJOR < 1
+#define FORMAT format
+#define FORMAT_KIND format
+#define FORMAT_KIND_UNDEF mkdnn_format_undef
+#define FORMAT_ANY mkldnn_any
+#define FORMAT_UNDEF mkldnn_undef
+#define DATA_UNDEF data_undef
+
+#define CHANGE_FORMAT                                                                              \
```

Comment:
wrap in do/whole(0), so we can do:
`CHANGE_FORMAT();` Same for other macros.

Reply:
why is do/while(0) better?